src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py

```diff
@@ -13,8 +13,8 @@
 from torch.fx import GraphModule
 
 from openvino.frontend import FrontEndManager
-from openvino.frontend.pytorch.fx_decoder import TorchFXPythonDecoder
-from openvino import Core, Type, PartialShape, serialize
+from openvino.frontend.pytorch.fx_decoder import TorchFXPythonDecoder, ExecuTorchPythonDecoder
+from openvino.runtime import Core, Type, PartialShape, serialize
```

Comment:
```suggestion
from openvino import Core, Type, PartialShape, serialize
```

`openvino.runtime` gets deprecated this release